### ISSUE TITLE
Incorrect text entry on Richmond's investigation for Stallion

### ISSUE THREAD

**Issue by onmyouji** [pyriell/gs2-bugfixes#1]
Hi,

I noticed this incorrect entry on Richmond's investigation (see screenshot). I'm not sure if it's caused by the patch, but if I remember correctly it wasn't like this before.

![suikoden ii usa -0026](https://user-images.githubusercontent.com/8424198/33226117-8f3594ba-d1b8-11e7-9539-c426183d9711.png)

**Comment by pyriell** [comment]
Huh, I wasn't aware that rune name appeared in any text that wasn't a copy of the rune list itself.  I'll probably have to sub in a whitespace character in those instances.  Otherwise it's more of a task for a translation/retranslation effort to ensure every occurrence is perfect.  In this case, the real solution would be to change the name and shift everything after that back one byte, but it looks like doing that might also lead to issues with fitting the text in the window properly.

**Comment by pyriell** [comment]
I searched all the places where the rune name appears, and the only one that isn't a copy of the rune list is this one.  The name is immediately followed by a newline character in the string, so formatting/space in the text box isn't an issue.  The process has been changed to shift the remaining text up, so this should be fixed by reapplying the patch.

I verified the change is working in the data files, but I haven't had an opportunity to play-test, so I'm closing this a bit provisionally for now.  I can't see any reason the fix shouldn't work just fine, but it'll take quite a few hours for me to confirm in-game.
